charts/gatekeeper/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: A Helm chart for Gatekeeper
 name: gatekeeper
 keywords:
   - open policy agent
-version: 3.7.0
```

Comment:
this will be updated automatically when we release next version. You can revert this change